packages/builder/src/builder.js

```diff
@@ -502,20 +508,19 @@ export default class Builder {
 
   watchClient() {
     const src = this.options.srcDir
+    const rGlob = dir => ['*', '**/*'].map(glob => r(src, `${dir}/${glob}.{${this.supportedExtensions.join(',')}}`))
```

Comment:
@pi0 Thoughts about this thing are welcome. This refactored well what was before in this `watchClient` method. For the rest of the code using `this.supportedExtensions`, I'm not sure how we can refactor it, any ideas or current state is ok to you ?

Reply:
Let's keep it as is for now. Nothing blocking :)